test/functional/viml/max_args_spec.lua

```diff
@@ -0,0 +1,29 @@
+local helpers = require('test.functional.helpers')
+
+local clear = helpers.clear
+local eq = helpers.eq
+local exc_exec = helpers.exc_exec
+
+describe('Up to MAX_FUNC_ARGS arguments are handled by', function()
```

Comment:
Consider renaming the test to `function_spec.lua`. (Unless it already exists, then these should be moved there)